libs/user/user.ts

```diff
@@ -454,6 +457,59 @@ class User {
     }
   }
 
+  /**
+   *
+   * @async
+   * @method importAndSaveCommitments should coverage the import commitments flow.
+   * - Should read and validate a file with commitments.
+   * - Verify if all the commitments read are of the ICommitment type (This verification is within readAndValidateFile function).
+   * - Verify if all the commitments belongs to the user compressedZkpPublicKey.
+   * - If all verifications pass, should send the commitments to the client to be saved
+   *  in the database.
+   * @param pathToExport the path to export the file.
+   * @param fileName the name of the file.
+   * @param compressedZkpPublicKey the key derivated from user mnemonic.
+   * @returns the enpoint response.data that is an json with a success message.
+   */
+  async importAndSaveCommitments(
+    pathToExport: string,
+    fileName: string,
+    compressedZkpPublicKey: string,
```

Comment:
Please pass these params in an `options` object - create an interface just like you did with the export `UserExportCommitments`.

But I can't find Joi validations for the export... Perhaps we can create a small issue for implementing both export/import validations with Joi and do this later :+1: